dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/FourEyeApproverActionlet.java

```diff
@@ -0,0 +1,166 @@
+package com.dotmarketing.portlets.workflows.actionlet;
+
+import static com.dotmarketing.portlets.workflows.util.WorkflowActionletUtil.getApproversFromHistory;
+import static com.dotmarketing.portlets.workflows.util.WorkflowActionletUtil.getParameterValue;
+import static com.dotmarketing.portlets.workflows.util.WorkflowActionletUtil.getUsersFromIds;
+
+import com.dotcms.util.ConversionUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.workflows.model.MultiUserReferenceParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionClassParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowHistory;
+import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
+import com.dotmarketing.portlets.workflows.util.WorkflowEmailUtil;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Sometimes, customers would like content to be published if a specific number of people approve
+ * their content. They are not particular as to which users, they just need a specific number of
+ * users to approve it before the content goes live. This actionlet enforces what is called the
+ * '4-Eyes' principle.
+ * <p>
+ * This workflow actionlet allows a user to specify the user IDs, email addresses, or role keys
+ * (i.e., all the users assigned to those roles) which will be in charge of approving the new
+ * content. If the users that approve the content are greater or equal to the specified number of
+ * minimum approvers, then the content will move on to the next actionlet in the workflow.
+ * Otherwise, an email will be sent to all users that haven't approved the content yet.
+ *
+ * @author Jose Castro
+ * @version 4.3.0
+ * @since Jan 3, 2018
+ */
+public class FourEyeApproverActionlet extends WorkFlowActionlet {
+
+    private static final long serialVersionUID = 1177885642438262884L;
+
+    private static final String ACTIONLET_NAME = "'4 Eye' Approval";
+    private static final String HOW_TO =
+            "This actionlet implements the '4 Eyes' principle for verifying new content. It takes a comma-separated list of user IDs, email addresses, or role keys "
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 165 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)